tests/Behat/Context/Transform/PriceContext.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+
+final class PriceContext implements Context
+{
+    /**
+     * @Transform /^"(\-)?((?:\d+\.)?\d+)"$/
+     */
+    public function getPriceFromString(string $sign, string $price): int
+    {
+        $this->validatePriceString($price);
+
+        $price = (int) round((float) $price * 100, 2);
+
+        if ('-' === $sign) {
+            $price *= -1;
+        }
+
+        return $price;
+    }
+
+    /** @throws \InvalidArgumentException */
+    private function validatePriceString(string $price): void
+    {
+        if (!(bool) preg_match('/^\d+(?:\.\d{1,2})?$/', $price)) {
```

Comment:
Can't we use this regular expression in `@Transform`?

Reply:
To be honest, I'm not sure, but I will check that